openmoc/plotter.py

```diff
@@ -14,6 +14,8 @@
 #         openmoc.plotter module.
 openmoc = ''
 
+print('whoa!!!')
```

Comment:
While the plotter is pretty awesome, I'm not sure we need to print this when it is included.